### PR TITLE
[APP] Atuin

### PR DESCRIPTION
**Pull request by cori**
(from the description.md)

---

### Making your shell magical

Sync, search and backup shell history with Atuin

[Atuin](https://atuin.sh) is an open-source Terminal User Interface (TUI) for your shell history.

### Features

1. Shell History Sync 
    - Sync your shell history to all of your machines, wherever they are
1. End-to-end Encryption 
    - All data is encrypted, and can only be read by you
1. Efficient Search
    - Search decades of shell history, and recall it in an instant. Atuin offers configurable full text or fuzzy search, filterable by host, directory, etc.
1. Data Import
    - Bring your existing history with you - Atuin supports importing from a wide variety of formats
1. Store Extra Context
    - Atuin stores extra context with your commands - working directory, exit code, and more!

### Self-hosted sync server

Atuin.sh offers a free, fully-encrypted option for storing your synced shell history as well as an option to self-host a sync server of your own. This is an easy-to-use package for the Atuin sync server component; the user interface is the TUI, which can be installed using many typical package-management tools, e.g.

- `brew install atuin`
- `apt install atuin`

or via `bash <(curl --proto '=https' --tlsv1.2 -sSf https://setup.atuin.sh)`.

---

regarding the MR

- there are two commits unrelated to adding Atuin
  - I gitignored the `.vscode` directory - maybe not doing so was intentional?
  - I couldn't see the link to the guide for adding an app - I didn't notice it in the markdown until I'd already been plugging at this for a bit, so I "fixed" it so I could see it

## Summary by CodeRabbit

- **Chores**
  - Updated `.gitignore` to exclude the `.vscode/` directory.

- **Documentation**
  - Updated the project README with the new documentation URL.
  - Added a description for the Atuin Server in the metadata section.

- **New Features**
  - Introduced the Atuin Server, featuring shell history synchronization and backup with encryption, efficient search, and data import capabilities.

### CODE REVIEW

**Comment by cori** [comment]
also unrelated; I see some other ... irregularities in other docs. from a practice standpoint are merge requests for details like that desirable? Better to isolate from other work? Should I isolate the unrelated stuff from this work into another request?

Also Atuin's server component feels a bit different than most of what's in Tipi right now. It's **super** useful to me to have it in here and I'd love to see more folks using Atuin, but that doesn't make it _right_ for Tipi, so let me know if it's a poor fit.

File: apps/atuin/metadata/description.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+1. Shell History Sync 
+    - Sync your shell history to all of your machines, wherever they are
+1. End-to-end Encryption 
+    - All data is encrypted, and can only be read by you
+1. Efficient Search
+    - Search decades of shell history, and recall it in an instant. Atuin offers configurable full text or fuzzy search, filterable by host, directory, etc.
+1. Data Import
+    - Bring your existing history with you - Atuin supports importing from a wide variety of formats
+1. Store Extra Context
+    - Atuin stores extra context with your commands - working directory, exit code, and more!
```
[the remark this answers is not in the document]
Comment (cori) [reply apps/atuin/metadata/description.md:22]: ooooh.

I disagree with this.

Markdown should auto-number, and using the autonumbering scheme saves time and effort when inserting items mid-list.

That said, not a hill i'm prepared to defend aside from the above....

**Comment by cori** [comment]
also, port 8888 is the default, and is free per https://docs.google.com/spreadsheets/d/13U8MNCOiQjR4XohNCl9-CrmqS8_CA-oFEy3wrnJBJks/edit#gid=0

**Comment by meienberger** [comment]
Thank you @cori here are [a few corrections](https://github.com/runtipi/runtipi-appstore/commit/29112c85204bd62d1f5d310967b0600f6d7e3199) I made before merging

**Pull request merged**